enlighten/BLEManager.py

```diff
@@ -102,19 +106,18 @@ def ble_discover_click(self, btn, device):
 
     def perform_connect(self, btn, device):
         log.debug(f"called to perform connect on btn {btn} with text {btn.text()}")
-        self.ble_device = BLEDevice(device, self.loop)
-        ok = self.controller_connect(self.ble_device)
+        #self.ble_device = BLEDevice(device, self.loop)
+        self.ble_device_id = DeviceID(label=f"BLE:{device.address}:{device.name}")
```

Comment:
I look forward to getting one of these units to test with :-)  (An actual BLE SiG, though I realize I could do with an RPi now...)